src/App.jsx

```diff
@@ -28,7 +41,7 @@ const App = () => {
     <>
       <h1>Todo List</h1>
       <AddTodoForm onAddTodo={addTodo} />
-      <p></p>
+      {isLoading ? <p>Loading list...</p> : null}
       <TodoList todoList={todoList} onRemoveTodo={removeTodo} />
```

Comment:
welldone Jeffery, i just need you to make a simole change, the tenary `isLoading` statement is used to show the `loading list...` text when the app is load and waiting for data or show the `Todolist` when the data is present so I suggest you do this

```suggestion
{isLoading ? <p>Loading list...</p> : 
      <TodoList todoList={todoList} onRemoveTodo={removeTodo} /> }
```